libtsuba/test/storage-format-version/v3-optional-topologies.cpp

```diff
@@ -325,6 +325,15 @@ TestGraphBasicRoundTrip(const std::string& rdg_name) {
   return katana::ResultSuccess();
 }
 
+katana::Result<void>
+Run(const std::string& rdg_str) {
```

Comment:
Big fan of wrapping all of the test cases in a `Run` func where we can use the usual `KATANA_CHECKED`